typescript/base/src/projects/ima/mainnet/MainnetImaInstance.ts

```diff
@@ -25,20 +37,34 @@ const contractManagerAbi = [
     }
 ];
 
+const contractExists = (
```

Comment:
Move this to `Instance` to avoid duplication?
This method does the same in each particular instance.